src/anndata/_core/aligned_mapping.py

```diff
@@ -175,9 +175,8 @@ def __setitem__(self, key: str, value: Value) -> None:
 
     def __delitem__(self, key: str) -> None:
         if key not in self:
-            raise KeyError(
-                "'{key!r}' not found in view of {self.attrname}"
-            )  # Make sure it exists before bothering with a copy
+            msg = f"'{key!r}' not found in view of {self.attrname}"
+            raise KeyError(msg)  # Make sure it exists before bothering with a copy
```

Comment:
I spotted this error: There was an `f` missing, so this error message was pretty useless